sdks/client-go/streaming_client_analytics.go

```diff
@@ -0,0 +1,7 @@
+package client
+
+// LogAnalyticsEvent submits analytics events using the client's configured AnalyticsCollector:
+func (c *StreamingClient) LogAnalyticsEvent(action string, other map[string]string) error {
+
+	return c.analyticsCollector.LogEvent(action, other, c.features)
```

Comment:
so this looks like you are supporting a single analytics collector?

and does c.features snapshot the events in case they change while the event logging is taking place? or is that intended for a future PR?

Reply:
Ah, you would like to have multiple analytics collectors going at once? I'm sure that can be done.

You're right about the snapshotting - that isn't there yet.

Reply:
OK I've make it copy the feature set before submitting events now